test/mutable/index.js

```diff
@@ -73,6 +79,32 @@ describe('Mutable Data', () => {
             })
     );
 
+    it('throws error if custom public is created with name not equal to 32 bytes', async () => {
```

Comment:
This seems a duplicate of this test: https://github.com/maidsafe/safe_app_nodejs/pull/208/files#diff-de3a919540b70d521ff29a18e30c207aR40